src/components/02-structure/05-navbar/navbar.njk

```diff
@@ -0,0 +1,82 @@
+<div class="sds-navbar border-bottom border-base-lighter">
+  <div class="sds-navbar__content">
+    <div class="grid-row">
+      <div
+        class="grid-col-12 tablet-lg:grid-col-auto display-flex flex-align-center margin-bottom-1 tablet-lg:margin-bottom-0">
+        <!-- Back Button -->
+        <button class="sds-button sds-button--circular">
+          {%- render '@icon', {prefix: 'fas', classes: 'fa-fw', icon:'chevron-left'} -%}
+        </button>
+
+        <!-- Page Title -->
+        <span class="font-heading-md text-semibold tablet-lg:width-mobile">
+          Title here
+        </span>
+      </div>
+      <div class="grid-col-12 tablet-lg:grid-col-fill display-flex">
+        <div class="tablet-lg:order-1">
+          <!-- Buttons -->
+          <button class="usa-button usa-button--secondary">Button</button>
```

Comment:
Why was it added in this ticket? Why do we have this and the Headers under Structure?
https://federalist-0ad5a602-ca98-4a7e-8d6e-d9ece7bc4cf8.app.cloud.gov/site/gsa/sam-styles/components/detail/header--default.html

Reply:
Well, the navbar was part (as is) of the header. At first this ticket included creating full layouts of pages, because some did not include the navbar I separated the navbar as is. During demo  it was realized that all of the header elements including the navbar should have option to make them accurate to the places where they are used. Christy decided this was out of scope so the have been removed until they can be refined in fractal to accept options and settings